dev/breeze/src/airflow_breeze/commands/testing_commands.py

```diff
@@ -1022,6 +1024,92 @@ def helm_tests(
     sys.exit(result.returncode)
 
 
+@group_for_testing.command(
+    name="openapi-tests",
+    help="Run open api tests.",
+    context_settings=dict(
+        ignore_unknown_options=True,
+        allow_extra_args=True,
+    ),
+)
+@option_install_airflow_python_client
```

Comment:
I think there is no need to have those two flags here. I can't imagine running "openapi-tests" command and NOT having those two flags set - basically "openapi-tests" should always set those to True - and we can hard-code them.

On the other hand, those options should be set as flags in "shell" command. The "shell" command is basically the same as `breeze` but it has way more flags/options and you can use it to enter breeze and emulating what other testing commands are doing without actually running the tests - which is very convenient if you want to iterate over tests in breeze.

That should allow the user to do this:

```
breeze shell --install-airflow-python-client --start-airflow-minimal-webserver-with-examples 
```

And then run the test manually with pytest or as script and iterate on it.

Reply:
Also LIKELY the `--start-airflow-minimal-webserver-with-examples` is a BIT too descriptive. Maybe `--start-webserver-with-examples`  would be a bit better ?

Reply:
Ah i see great point i will update them :)